packages/validators-util/src/isValidName/isValidName.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { isValidName } from "./isValidName";
+
+describe("isValidName", () => {
+    it("should return true if no illegal chars", () => {
+        expect(isValidName("Kenneth")).toBe(true);
+        expect(isValidName("Kenneth Apeland")).toBe(true);
+        expect(isValidName("Rávna Ròverdatter")).toBe(true);
+        expect(isValidName("Jökull Guðrún")).toBe(true);
+    });
+
+    it("should return false if contains Punctuation, Symbols or Digits", () => {
+        expect(isValidName("sd@sd")).toBe(false);
+        expect(isValidName("Har du det bra?")).toBe(false);
+        expect(isValidName("Nei.")).toBe(false);
+        expect(isValidName("{asdads}")).toBe(false);
+        expect(isValidName("Heisann (sveisann)")).toBe(false);
+        expect(isValidName("Adam's matkasse")).toBe(false);
```

Comment:
_Mike O'Brien has left the chat_

Reply:
Ikke for å være supervanskelig altså, men akkurat det tegnet kan godt være tillatt 😄

Reply:
Er nå lov og la til en kommentar i regex om at den er lov, pluss flyttet testen